db/migrate/20150416231106_add_source_to_webhook.rb

```diff
@@ -0,0 +1,6 @@
+class AddSourceToWebhook < ActiveRecord::Migration
+  def change
+    add_column :webhooks, :source, :string, default: 'any_ci'
+    Webhook.update_all source: 'any_ci'
```

Comment:
I think if you add the non-null contraint, then the default will be added to all old rows. But I'm not 100% sure across the 3 DBs we support.